### PR TITLE
Pandas.io.formats.style.Styler docstring PR02

### PR DESCRIPTION
**Pull request by Genza999**
Solves:

- Unknown parameters {kwargs} in apply method in Styler class
- Unknown parameters {kwargs} in applymap method in Styler class
- Unknown parameters {kwargs} in where method in Styler class
- Unknown parameters {low, high}  in background_gradient method in Styler class
- Unknown parameters {kwargs} in set_properties method in Styler class
- Unknown parameters {*args, **kwargs :} in pipe method in Styler class

for issues:

pandas.io.formats.style.Styler.apply: Unknown parameters {kwargs}
pandas.io.formats.style.Styler.applymap: Unknown parameters {kwargs}
pandas.io.formats.style.Styler.where: Unknown parameters {kwargs}
pandas.io.formats.style.Styler.set_properties: Unknown parameters {kwargs}
pandas.io.formats.style.Styler.pipe: Unknown parameters {*args, **kwargs :}
pandas.io.formats.style.Styler.background_gradient: Unknown parameters {low, high}

all in #27976

### CODE REVIEW

File: pandas/io/formats/style.py [hunk 1]
```diff
@@ -1350,8 +1352,11 @@ def pipe(self, func, *args, **kwargs):
             Function to apply to the Styler.  Alternatively, a
             ``(callable, keyword)`` tuple where ``keyword`` is a string
             indicating the keyword of ``callable`` that expects the Styler.
-        *args, **kwargs :
+        *args : iterable, optional
             Arguments passed to `func`.
+        **kwargs : dict, optional
+            A dictionary of keyword arguments passed into ``func``.
+
```
Comment (jbrockmendel) [inline pandas/io/formats/style.py]: Looks like there are too many newlines here

Comment (Genza999) [reply pandas/io/formats/style.py]: Thanks @jbrockmendel,  I have removed the extra new line that was added.

**Comment by jreback** [comment]
thanks @Genza999 this doesn't fully close #27976 right?

**Comment by Genza999** [comment]
> thanks @Genza999 this doesn't fully close #27976 right?

No it doesnt @jreback , it just covers a few of the issues.

**Pull request merged**